TutorialApplication/CreateEmployeeViewController.m

```diff
@@ -9,8 +9,31 @@
 #import "CreateEmployeeViewController.h"
 #import "Employee.h"
 #import "DatabaseController.h"
+#import "HSDatePickerViewController.h"
 
-@implementation CreateEmployeeViewController
+@interface CreateEmployeeViewController()<HSDatePickerViewControllerDelegate>
+
+@end
+
+@implementation CreateEmployeeViewController 
+
+-(void)hsDatePickerPickedDate:(NSDate *)date
+{
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    [dateFormatter setDateFormat:@"yyyy-MM-dd"];
+    self.dateOfBirthTextField.text = [dateFormatter stringFromDate:date];
+}
+
+-(BOOL)textFieldShouldBeginEditing:(UITextField *)textField
+{
+    HSDatePickerViewController *hsdpvc = [[HSDatePickerViewController alloc]init];
```

Comment:
You shouldn't do any actions in a method, which returns boolean value. Move this logic to didBeginEditing